crates/biome_cli/tests/commands/format.rs

```diff
@@ -180,14 +180,10 @@ const APPLY_ATTRIBUTE_POSITION_AFTER: &str = r#"<Foo
 </Foo>;
 "#;
 
-// Without this, Test (windows-latest) fails with: `warning: constant `DEFAULT_CONFIGURATION_BEFORE` is never used`
-#[allow(dead_code)]
 const DEFAULT_CONFIGURATION_BEFORE: &str = r#"function f() {
     return { a, b }
   }"#;
 
-// Without this, Test (windows-latest) fails with: `warning: constant `DEFAULT_CONFIGURATION_AFTER` is never used`
-#[allow(dead_code)]
 const DEFAULT_CONFIGURATION_AFTER: &str = "function f() {
```

Comment:
Looks like we lost these comments - are they still relevant?

Reply:
The Windows test still passed with both lines removed, so it seemed not necessary.